pkg/controller/cloud/node_controller_test.go

```diff
@@ -692,7 +690,7 @@ func Test_AddCloudNode(t *testing.T) {
 				recorder:                  eventBroadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: "cloud-node-controller"}),
 				nodeStatusUpdateFrequency: 1 * time.Second,
 			}
-			eventBroadcaster.StartLogging(klog.Infof)
+			eventBroadcaster.StartStructuredLogging(0)
```

Comment:
I think there was an earlier mistake here. I'd expect tests to start logging to t.Logf

Reply:
Thank you for your comment.
I fix this test.